pr-check.sh

```diff
@@ -18,6 +18,34 @@
 
 set -e
 
+# Check whether we are running on Princeton's della compute cluster.
+is_della=false
```

Comment:
This script has a lot of carve outs for Princeton's set up, which isn't a versatile solution for other groups nor will it future proof Brainiak within Princeton (e.g., if Della changes). Is there a way instead to make a script that replaces this one with the della-specific commands that are needed?